styleguide/screens/Patterns/TabBar/TabBar.js

```diff
@@ -66,7 +66,6 @@ const BadgeDocumentation = () => (
           </TabBarItem>
           <TabBarItem
             href="#notification"
-            active
```

Comment:
How come you removed this?

Reply:
I could be wrong but is that not how they are intended to be used? i.e., only one tab can be active at one time. More than one active tab item throws a warning and breaks the tests.

Reply:
Turns out it's really hard to tell from the diff that you're not removing the one correct usage of it 😅